src/proteus/dswx_hls.py

```diff
@@ -3155,17 +3155,21 @@ def _antimeridian_crossing_requires_special_handling(
 
     '''
 
-    # The dateline crossing by the tile is tested by the condition:
-    #     `tile_min_x < 180 and tile_max_x >= 180`
-    # The ancillary input only requires reprojection if it is in
-    # geographic coordinates and if its longitude domain is represented
-    # within the [-180, +180] range, verified by the test `min_x < -170`,
-    # rather than the [0, +360] interval. There's no specific reason why
-    # -170 is used. It could be -160, or even 0.
+    # Flag to indicate if the if the MGRS tile crosses the antimeridian.
+    flag_tile_crosses_antimeridian = tile_min_x < 180 and tile_max_x >= 180
 
+    # Flag to test if the ancillary input file is in geographic
+    # coordinates and if its longitude domain is represented
+    # within the [-180, +180] range, rather than the [0, +360] interval.
+    # This is verified by the test `min_x < -170`,
```

Comment:
Suggestion:
```
    # This is verified by the test `min_x < -170`. There's no specific reason why
    # -170 is used. It could be -160, or even 0.
 ```